dm/dm/ctl/master/binlog.go

```diff
@@ -90,11 +125,23 @@ func newBinlogRevertCmd() *cobra.Command {
 // FIXME: implement this later.
 func newBinlogInjectCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:    "inject <task-name> <inject-sql1> <inject-sql2>...",
-		Short:  "inject the current error event or a specific binlog position (binlog-pos) ddl event with some ddls",
-		Hidden: true,
+		Use:   "inject <task-name> <inject-sql1> <inject-sql2>...",
+		Short: "inject the current error event or a specific binlog position (binlog-pos) ddl event with some ddls",
```

Comment:
```suggestion
		Short: "inject the current error event or a specific binlog position (binlog-pos) with some ddls",
```

do you mean this?

Reply:
yes, i will fix inject and replace desc.